bridge/client/app/_components/ktb-deployment-list/ktb-deployment-list.component.html

```diff
@@ -22,5 +22,5 @@
     </dt-cell>
   </ng-container>
 
-  <dt-row *dtRowDef="let row; columns: ['version', 'stages']" (click)="selectDeployment(row)" [class.active]="selectedDeployment?.shkeptncontext === row.shkeptncontext"></dt-row>
+  <dt-row *dtRowDef="let row; columns: ['version', 'stages']" (click)="selectDeployment(row)" [class.active]="selectedDeployment?.shkeptncontext === toDeployment(row).shkeptncontext"></dt-row>
```

Comment:
Why this change?

Reply:
To have a type-check for `row`, because row has type `any` but is `Deployment`. If there would be a change of the property name `shkeptncontext` of `Deployment`, the IDE wouldn't find it.